index.js

```diff
@@ -767,6 +767,10 @@ var Client = module.exports = function(config) {
             options.rejectUnauthorized = this.config.rejectUnauthorized;
 
         if (this.debug) {
+            // options.uri can contain access_token
+            if (options.uri) {
+                options.uri = options.uri.replace(/access_token=(.*)/g,'access_token=<TOKEN_REMOVED>');
```

Comment:
@abtris @spaze __Blocking - this won't work.__

Because you've just removed the `access_token` from `options` -- those options are (few lines below) passed to `request` library.